### PR TITLE
Added new extension function for checking for permissions

### PR DESCRIPTION
**Pull request by acefalobi**

### CODE REVIEW

File: utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt [hunk 1]
```diff
@@ -48,4 +51,33 @@ class ContextUtilsTest {
...
+        val expectedResult1 = true
+
+        val actualResult2 = app.hasPermissions(
+            Manifest.permission.CAMERA,
+            Manifest.permission.ACCESS_FINE_LOCATION
+        )
+        val expectedResult2 = false
+
+        val actualResult3 = app.hasPermissions(
+            Manifest.permission.CAMERA,
```
[the remark this answers is not in the document]
Comment (acefalobi) [reply utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt:74]: hm true

[the remark this answers is not in the document]
Comment (acefalobi) [reply utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt:81]: fixed

File: utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt [hunk 2]
```diff
@@ -48,4 +51,74 @@ class ContextUtilsTest {
...
+
+        val actualResult = app.hasPermissions(Manifest.permission.CAMERA)
+
+        val expectedResult = true
+
+        assert(actualResult == expectedResult)
+    }
+
+    @Test
+    fun hasPermissions_truePositiveMultiple() {
```
Comment (Ezike) [inline utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt:72]: Feel like the test names are a bit confusing

Comment (Ezike) [inline utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt:72]: Maybe the long test name notation will be easier to read..

`check that hasPermissions does stuff when stuff happens `

Comment (acefalobi) [reply utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt:72]: i think they're more technical and precise than sentence names

https://en.wikipedia.org/wiki/False_positives_and_false_negatives

**Pull request merged**